src/styles/icon.scss

```diff
@@ -0,0 +1,19 @@
+
+.product-card__image {
+  width: 160px;
+  height: 134px;
+  margin: auto 3px;
+}
+
+.rating-stars__star {
+  background-image: url(../images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+
+  &:nth-child(-n + 4) {
+    background-image: url(../images/star-active.svg);
+  }
```

Comment:
Consider keeping the functionality for the stars block from the previous task

Reply:
If I understand you correctly  :-)
Thank you

Reply:
> everything is so right!? Thanks